wp-cache-phase1.php

```diff
@@ -506,7 +506,7 @@ function wp_cache_debug( $message, $level = 1 ) {
 		return false;
 
 	// Log message: Date URI Message
-	$log_messase = date('H:i:s') . " {$_SERVER['REQUEST_URI']} {$message}\n\r";
+	$log_message = date('H:i:s') . " {$_SERVER['REQUEST_URI']} {$message}\n\r";
```

Comment:
Noting that this fix was made already in 0d392b07b71cbf21bb5ac71e6d52317f3fab2cc3